Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+// vim: ft=groovy
+parallel(
+    cloud: {
+        node('cloud') {
+            deleteDir()
+            checkout scm
+            stage('CLOUD: Build a Factory') {
+                sh './factory prepare-cloud-image'
+                sh 'cd images/cloud-x86_64 && tar c * | xz > ../../cloud-x86_64-image.tar.xz'
```

Comment:
next line is `xz -0`, this one doesn't. Let's be consistent with the other repos and use `xz -1` everythere.

Reply:
Let's use `-0`, it's the quickest, and still better than gzip.